### PR TITLE
[1476] copy course length and salary content

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Copying course length and salary content

### Changes proposed in this pull request
Add "Copy from" select to enable users to copy content from another course within their org

### Guidance to review
Example - `/organisations/2AT/courses/3CXD/salary`

### CODE REVIEW

**Comment by dankmitchell** [comment]
[[S] Add sidebar with Copy functionality to "Course length and salary" edit page](https://trello.com/c/a8btME1w/1476-s-add-sidebar-with-copy-functionality-to-course-length-and-salary-edit-page)

File: app/views/courses/salary.html.erb [hunk 1]
```diff
@@ -4,6 +4,26 @@
   <%= govuk_back_link_to(provider_course_path(course.provider.provider_code, course.course_code)) %>
 <% end %>
 
+<% if params[:copy_from].present? %>
+  <div class="govuk-warning-summary" aria-labelledby="warning-summary-heading" tabindex="-1" data-module="error-summary" data-copy-course="warning" role="alert">
```
Comment (defong) [inline app/views/courses/salary.html.erb]: data-module="error-summary"

?

Comment (dankmitchell) [reply app/views/courses/salary.html.erb]: Looks odd but it means we can re-use the functionality from this module out of govuk-frontend https://github.com/alphagov/govuk-frontend/blob/master/src/components/error-summary/error-summary.js

File: app/views/courses/salary.html.erb [hunk 2]
```diff
@@ -4,6 +4,26 @@
...
+    <h2 class="govuk-heading govuk-warning-summary__title" id="warning-summary-heading">
+      Your changes are not yet saved
+    </h2>
+    <div class="govuk-warning-summary__body">
+      <p class="govuk-body">
+        We’ve copied these fields from <%= @source_course.name %> (<%= @source_course.course_code %>):
+      </p>
+      <ul class="govuk-list">
+        <li><%= link_to 'Course length', '#course_length_wrapper', class: 'govuk-link' %></li>
```
[the remark this answers is not in the document]
Comment (dankmitchell) [reply app/views/courses/salary.html.erb]: Good shout @defong. I'll merge this then to a follow up PR 👍

**Pull request merged**